assembl/views/__init__.py

```diff
@@ -559,8 +563,12 @@ def get_description(request):
             locale = discussion.preferences['preferred_locales'][0]
             if locale in objectives_dict:
                 return adapt_to_html_content(objectives_dict.get(opengraph_locale, objectives_dict[locale]))
+            elif len(objectives_dict) != 0:
```

Comment:
Hm, okay, you could traverse the the list of `preferred_locales` and see which key is in the `objectives_dict` before going to pick the first key from the dictionary. Extend this logic to the other function as well 👍